site/specs/numbers.json

```diff
@@ -16362,159 +16363,145 @@
           },
           {
             "name": "accountId",
-            "description": "<p>The internal account id assigned to the customer that \"owns\" the TN</p>\n",
+            "description": "The internal account id assigned to the customer that \"owns\" the TN.",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "integer",
-              "default": null
+              "type": "integer"
             },
             "example": 5000125
           },
           {
-            "name": "fullNumber",
```

Comment:
I see `fullNumber` removed here and replaced with `city` but it wasnt added back elsewhere in the parameters - is this not a working parameter for the endpoint?

Reply:
I sorted query param as in the original API-doc. The ```fullnumber``` goes after the ```city.```.

Reply:
Ah - i see it on 16385. Missed it at first glance. Looks good to me!